pkg/common/cluster/healthchecks/clusterversionoperator.go

```diff
@@ -37,3 +42,29 @@ func CheckCVOReadiness(configClient configclient.ConfigV1Interface, logger *log.
 
 	return success, nil
 }
+
+// CheckMachinesObjectState lists all openshift machines and validates that they are "Running"
+func CheckMachinesObjectState(dynamicClient dynamic.Interface, logger *log.Logger) (bool, error) {
+	mc := dynamicClient.Resource(schema.GroupVersionResource{Group: "machine.openshift.io", Resource: "machines", Version: "v1beta1"})
+	obj, err := mc.List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return false, err
+	}
+	var runningPhase string
+	runningPhase = "Running"
```

Comment:
Can we one line this? `var runningPhase string = "Running"`?

Reply:
Derp yeah.